bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClient.java

```diff
@@ -213,10 +213,11 @@ private Job copyData(
       TableId destinationTableId,
       JobInfo.WriteDisposition writeDisposition) {
     String queryFormat = "SELECT * FROM `%s`";
+    String temporaryTableName = fullTableName(sourceTableId);
```

Comment:
Changes to the copyData() function are not needed.

Reply:
it is the same method which has been shifted into here since we will have two `sqlFromFormat` methods one for `SELECT` and the other for `MERGE`.  Since the format methods are closely tied to the queries, I have dropped them and made changes to the caller itself.